compiler/luci/pass/src/ResolveFormerCustomOpPass.test.cpp

```diff
@@ -16,11 +16,136 @@
 
 #include "luci/Pass/ResolveFormerCustomOpPass.h"
 
```

Comment:
The related content of this PR is here. (https://github.com/Samsung/ONE/pull/12318#discussion_r1436660933)